components/floatingactionbutton/index.css

```diff
@@ -0,0 +1,125 @@
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+.spectrum-FloatingActionButton {
+  --spectrum-floating-action-button-size: var(--spectrum-component-height-200);
+  --spectrum-floating-action-button-icon-size: var(--spectrum-workflow-icon-size-200);
+  --spectrum-floating-action-button-padding: var(--spectrum-component-pill-edge-to-visual-only-200);
+  --spectrum-floating-action-button-margin: var(--spectrum-spacing-200);
+  --spectrum-floating-action-button-drop-shadow-x: var(--spectrum-drop-shadow-x);
+  --spectrum-floating-action-button-drop-shadow-color: var(--spectrum-floating-action-button-shadow-color); 
+
+  --spectrum-floating-action-button-focus-ring-width: var(--spectrum-focus-indicator-thickness);
+  --spectrum-floating-action-button-focus-ring-gap: var(--spectrum-focus-indicator-gap);
+  --spectrum-floating-action-button-focus-ring-color: var(--spectrum-focus-indicator-color);
+
+  --spectrum-floating-action-button-background-color: var(--spectrum-accent-background-color-default);
+  --spectrum-floating-action-button-background-color-hover: var(--spectrum-accent-background-color-hover);
+  --spectrum-floating-action-button-background-color-down: var(--spectrum-accent-background-color-down);
+  --spectrum-floating-action-button-background-color-key-focus: var(--spectrum-accent-background-color-key-focus);
+  --spectrum-floating-action-button-icon-color: var(--spectrum-white);
+  --spectrum-floating-action-button-icon-color-hover: var(--spectrum-white);
+  --spectrum-floating-action-button-icon-color-down: var(--spectrum-white);
+  --spectrum-floating-action-button-icon-color-key-focus: var(--spectrum-white);
+
+  &--secondary {
+    --spectrum-floating-action-button-background-color: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-background-color-hover: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-background-color-down: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-background-color-key-focus: var(--spectrum-background-layer-2-color);
+    --spectrum-floating-action-button-icon-color: var(--spectrum-neutral-subdued-content-color-default);
+    --spectrum-floating-action-button-icon-color-hover: var(--spectrum-neutral-subdued-content-color-hover);
+    --spectrum-floating-action-button-icon-color-down: var(--spectrum-neutral-subdued-content-color-down);
+    --spectrum-floating-action-button-icon-color-key-focus: var(--spectrum-neutral-subdued-content-color-key-focus);
+  }
+
+}
+
+.spectrum-FloatingActionButton {
+  cursor: pointer;
+  block-size: var(--mod-floating-action-button-size, var(--spectrum-floating-action-button-size));
+  inline-size: var(--mod-floating-action-button-size, var(--spectrum-floating-action-button-size));
+  border-radius: 50%;
```

Comment:
Should we add a `--spectrum-floating-action-button-border-radius` custom property and use it here and in the focus indicator? I could see someone wanting access to a `--mod-floating-action-button-border-radius`. I suppose it also could be done without adding a property too, e.g. `border-radius: var(--mod-floating-action-button-border-radius, 50%);`

Reply:
Yeah I can add that in. These buttons are supposed to be perfect circles but since we're going super customizable I should probably add it just in case.